tests/Integrations/Curl/CurlIntegrationTest.php

```diff
@@ -46,6 +46,7 @@ public function tearDown()
     {
         parent::tearDown();
         putenv('DD_CURL_ANALYTICS_ENABLED');
+        putenv('DD_TRACE_HTTP_CLIENT_SPLIT_BY_DOMAIN');
```

Comment:
I don't see where this impacted the tests. Why was this added?

Reply:
The `CurlIntegrationTest.php` sets `DD_TRACE_HTTP_CLIENT_SPLIT_BY_DOMAIN` in a test and didn't clean up. If we ran `CurlIntegrationTest.php` by itself it was fine, and `CurlSandboxTest.php` by itself it was fine, but if we ran them one after another then we had a problem. So we clean it up in teardown in case the test fails that this is still unset and won't affect other tests (cascading false failures are annoying).